portfolio/src/main/webapp/index.html

```diff
@@ -195,9 +195,20 @@ <h2 style="color:FloralWhite; background-color:green;">
 
     <div id="content">
       <h2 style="color:FloralWhite; background-color:red;">Comments:</h2>
-      <p style="text-align:center">Click here to see some comments!</p>
+      <p style="text-align:center">Thank you for your time! Feel free to leave a comment or click on the 
+          button at the bottom of the page to see some other comments!
+      </p>
+
+      <form action="/data" method="POST">
+      <p>Enter your comment: </p>
+      <textarea name="comment-input">New Comment</textarea>
+      <br/><br/>  
+      <input type="submit" />
+      <br/><br/>
+      </form>
+
       <div style="border: 1px solid">
-      <button onclick="getServerComments()">Comments</button>
+        <button onclick="getServerComments()">Comments</button>
```

Comment:
_nitpick_ - remove indentation (this is admittedly minor, but as these keep adding up, the code can start to look messy)

Reply:
Yes! I also dislike these but I am having trouble spotting them on the editor for some reason